extractFeature.py

```diff
@@ -0,0 +1,44 @@
+#============================
+#PACKAGES
+#============================
+import sys
+import json
+import time
+import traceback
+from helpers import extractAAC,log,readFile
+
+
+#============================
+#MAIN
+#============================
+def main():
+
+    #Extracting feature from Dataset files
+    try:
+        #Logging start of Program
+        feature = sys.argv[1]
+        dataset = sys.argv[2]
```

Comment:
You should use `argparse` to parse your arguments. It would also allow to specify that dataset has to be a file, and that feature can only take specific values.